README.md

```diff
@@ -374,32 +374,34 @@ for await (let b of resp) {
 ```
 note: to get the date of response samples you could do this `console.log(new Date(resp['AAPL'][0].startEpochTime*1000))`
 
-#### Last trade
+#### Get the Latest trade
```

Comment:
I would rename this to `Latest Trade` or `latest trade`. Same for the latest quote.